pkgs/development/tools/rust/cargo-flash/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkg-config, rustfmt, libusb1 }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-flash";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "probe-rs";
+    repo = pname;
+    rev = "v" + version; 
+    sha256 = "01zyxbykzbqfy6w0cmhhq10l1qqf4aqxvh4jngj1lqba5r9d7jsc";
+  };
+
+  cargoPatches = [ ./cargo-lock.patch ];
+  cargoSha256 = "0fpcglng00vfx5643y2dcjik29vl2rz9vv8dx67gnpbkdrhpfxcx";
+
+  nativeBuildInputs = [ pkg-config rustfmt ];
+  buildInputs = [ libusb1.dev ];
+
+  doCheck = false;
```

Comment:
Guessing the tests have to be skipped because they try to do IO?

Reply:
Worse, this is an artifact of my habit of copying build scripts to make new pkgs. The test succeeds, at least on linux x86_64